src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -53608,6 +53608,26 @@ static void F3(object? x3, object y3)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "z2").WithLocation(22, 9));
         }
 
+        [Fact]
+        public void Lambda_20()
```

Comment:
Would this have failed before?

Reply:
> Would this have failed before?

No. But it was a test case we were missing.

---
In reply to: [560571260](https://github.com/dotnet/roslyn/pull/50417#discussion_r560571260) [](ancestors = 560571260)